bigtable/admin/table_admin.cc

```diff
@@ -97,32 +81,12 @@ std::vector<::google::bigtable::admin::v2::Table> TableAdmin::ListTables(
 
 ::google::bigtable::admin::v2::Table TableAdmin::GetTable(
     std::string table_id, ::google::bigtable::admin::v2::Table::View view) {
-  // Copy the policies in effect for the operation.
```

Comment:
I think you can also delete the same lines in `CreateTable()` but I can't leave a comment there for some reason.

Reply:
@coryan — this reminds me, is there a flag (or a static analysis) for either GCC or Clang to flag dead code within functions, e.g., local vars that are obviously unused and hence point to an oversight or error?

Reply:
> I think you can also delete the same lines in `CreateTable()` but I can't leave a comment
> there for some reason.

Done.  Probably cannot comment on code that was not touched, which is weird.

As to unused variables and dead code: I think `-Wall -Werror` already catches most of them, in this case I defeated the analysis by calling virtual functions and virtual destructors <shame>.  I do not know of any tool that will call that "dead code".